### ISSUE TITLE
HTML, CSS, JS Learnings 

### ISSUE THREAD

**Issue by vikramvi** [vikramvi/vanillawebprojects#14]
[Repo Link](https://github.com/vikramvi/vanillawebprojects/tree/master/lyrics-search)

1. add HTML
    - with "dummy data" as webpage is suppose to show info from API

2. add CSS
    - style as per design with "dummy data"

3.  add JavaScript
     - get all DOM elements
     - write user actions / results in plain English
     - add functions ( update DOM, get data, submit data, other user actions )
     - add event listeners
         - vice versa above 2 steps 
     - fetch call ( .then() vs async await )
     - add "HTML Custom Attributes" at run time
     - create new HTML container element & child elements from API data with  :  result.innerHTML + map + join
     - handle CORS bug during subsequent fetch calls

**Comment by jaypansuriya104** [comment]
Hi , I am interested to contribution ! can you assign me ??
